time start_time
uint8 motion_mode
float32[] coef_x
float32[] coef_y
float32[] duration
